src/twisted/python/_release.py

```diff
@@ -39,14 +40,50 @@
 
 
 def runCommand(args, **kwargs):
-    """Execute a vector of arguments.
+    """
+    Execute a vector of arguments.
 
     This is a wrapper around L{subprocess.check_output}, so it takes
     the same arguments as L{subprocess.Popen} with one difference: all
     arguments after the vector must be keyword arguments.
     """
     kwargs['stderr'] = STDOUT
-    return check_output(args, **kwargs)
+    output = check_output(args, **kwargs)
+    if not isinstance(output, str):
+        output = output.decode("utf-8")
+    return output
+
+
+
+def lenientSetStrContent(fileObj, content, encoding="utf-8"):
+    """
+    Write out C{content} as L{bytes} to a L{FilePath}.
+
+    @param fileObj: file to write to
+    @type fileObj: L{FilePath}
+    @param content: content to write to file
+    @type content: L{str} or L{bytes}
+    @param encoding: string encoding
+    """
+    if not isinstance(content, bytes):
+        content = content.encode(encoding)
+    fileObj.setContent(content)
+
+
+
+def getStrContent(fileObj, encoding="utf-8"):
+    """
+    Get the content from a L{FilePath} and return it as L{str}.
```

Comment:
Since this is designed specifically to return it as a `str` as opposed to `bytes` or `unicode`, it should use the disambiguating prefix "native".  Older code that is confused about text/bytes often documents types as `str` and that's sometimes vague when it comes to how other cross-version code should expect to consume it.